examples/cargo_without_cmake/Cargo.toml

```diff
@@ -7,7 +7,7 @@
 # ANCHOR: book_cargo_toml_no_cmake
 [package]
 name = "qml-minimal-no-cmake"
-version = "0.3.0"
+version = "0.1.0"
```

Comment:
our examples should just be `0.1.0` so we don't need to update the version in every release, as they aren't published to crates.io. Hence the change here.